src/UnitTests/AdvancedConfiguration.cs

```diff
@@ -0,0 +1,160 @@
+using System;
```

Comment:
Can you please rename this file/class to CustomValidations? And move the existing test here.

Reply:
I renamed it

Reply:
https://github.com/AutoMapper/AutoMapper/blob/7a00700de61cd3234b6a6eb4bf0f6bbe402369b1/src/UnitTests/ConfigurationValidation.cs#L9 belongs in this file too.

Reply:
This test is moved to the new CustomValidations file. I have searched for all usages of cfg.Advanced.Validator in test files, There is no one left.